client/src/App.css

```diff
@@ -76,3 +76,8 @@ footer {
   z-index: 10;
   border-radius: 0%;
 }
+
+.glass {
+  background-color: rgba(255, 255, 255, 0.192);
```

Comment:
Instead of hardcoding this color here, we should be able to use one of the colors like `glass-grey` or `dark-glass-grey` we added in the tailwind config file, and reference them directly in the `Glass` component

Reply:
Nothing happens with the backdrop-filter.